### ISSUE TITLE
Index error : The table does not have the specified index: Losses

### ISSUE THREAD

**Issue by benbellity** [awslabs/aws-sdk-ios-samples#58]
Hello, 
First of all, I'm a new AWS user so sorry for the question, which might probably be obvious to you.
I managed to run the DynamoDB swift project and generating the test data worked fine.
But when I went to the search screen, I get the following error when I press search : 

Error: Optional(Error Domain=com.amazonaws.AWSDynamoDBErrorDomain Code=0 "(null)" UserInfo={__type=com.amazon.coral.validate#ValidationException, message=The table does not have the specified index: Losses})

btw, I get the same error with Wins and with TopScore.
So, basically, my question is : Did I do anything wrong to get this error message ? 
(fyi I just created the table in dynamoDB by indicating that the key is GameTitle and defining UserId as the sort key)
Should I declare the index in the DynamoDB console ?

Thank you in advance for your help.
Best regards
Benjamin

**Comment by yosuke-matsuda** [comment]
The sample app automatically creates the appropriate table. When you manually create one, you should take a look at [how the app is creating a table](https://github.com/awslabs/aws-sdk-ios-samples/blob/master/DynamoDBObjectMapper-Sample/Objective-C/DynamoDBSample/DDBDynamoDBManager.m#L31) and copy the scheme.
